starport/templates/typed/list/stargate/messages/x/{{moduleName}}/simulation/{{typeName}}.go.plush

```diff
@@ -0,0 +1,116 @@
+package simulation
+
+import (
+	"math/rand"
+
+	"<%= ModulePath %>/x/<%= ModuleName %>/keeper"
+	"<%= ModulePath %>/x/<%= ModuleName %>/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+)
+
+func SimulateMsgCreate<%= TypeName.UpperCamel %>(
+	ak types.AccountKeeper,
+	bk types.BankKeeper,
+	k keeper.Keeper,
+) simtypes.Operation {
+	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simtypes.Account, chainID string,
+	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+		simAccount, _ := simtypes.RandomAcc(r, accs)
+
+		msg := &types.MsgCreate<%= TypeName.UpperCamel %>{
+			<%= MsgSigner.UpperCamel %>: simAccount.Address.String(),
+		}
+
+		txCtx := simulation.OperationInput{
+			R:               r,
+			App:             app,
+			TxGen:           simappparams.MakeTestEncodingConfig().TxConfig,
+			Cdc:             nil,
+			Msg:             msg,
+			MsgType:         msg.Type(),
+			Context:         ctx,
+			SimAccount:      simAccount,
+			ModuleName:      types.ModuleName,
+			CoinsSpentInMsg: sdk.NewCoins(),
+			AccountKeeper:   ak,
+			Bankkeeper:      bk,
+		}
+		return simulation.GenAndDeliverTxWithRandFees(txCtx)
+	}
+}
+
+func SimulateMsgUpdate<%= TypeName.UpperCamel %>(
+	ak types.AccountKeeper,
+	bk types.BankKeeper,
+	k keeper.Keeper,
+) simtypes.Operation {
+	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simtypes.Account, chainID string,
+	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+		simAccount, _ := simtypes.RandomAcc(r, accs)
+
+		<%= TypeName.LowerCamel %>ID := k.Append<%= TypeName.UpperCamel %>(ctx, types.<%= TypeName.UpperCamel %>{
```

Comment:
General comment: we should not use internal keeper writing method for simulation since it's outside of the lifecycle of an app.

CRUD messages for a type should be generally only used for learning purpose so I think we can use these message to general example of what simulation operation should look like.
So in this case we would have:
- Create: create a type with random values
- Update: search for a sim account with a created values and update them
- Delete: search for a sim account with a value and delete it

This may not be trivial to implement so maybe this could be implemented in another PR and we keep `NoMsg` for now

Reply:
Yeah, it makes sense. I can make a quick fix to avoid inserting directly by the keeper. It's an easy change